index.html

```diff
@@ -2065,9 +2065,18 @@
       <section>
         <h3>Highly Dynamic Information</h3>
 
-        <p class="issue">
-Time periods should be shorter for highly dynamic information.
+        <p>
+When verifiable <a>credentials</a> are issued for highly dynamic information,
+implementers should ensure that the expiration times for the credential are
+set appropriately. Expiration periods that are longer than the timeframe
+where the credential is valid may create exploitable security vulnerabilities.
+Expiration periods that are shorter than the timeframe where the information
+expressed by the credential is valid create a burden on <a>holders</a> and
+<a>verifiers</a>. It is therefore important to set validity periods for
+credentials that are appropriate to the use case and the expected lifetime
+for the information contained in the credential.
```

Comment:
Isn't this in direct contradiction to the recommendation to use short-lived credentials?

Reply:
Yes, perhaps we need more specific recommendations or a sliding scale -- or just a note that "if credentials are too short-lived the drawbacks are X and if they are too long-lived the drawbacks are Y." Followed by: "Use technology T or policy P for use cases where you can't compromise on either."

Reply:
Can we raise a new issue for this? I agree with both of you, but I feel like we need to refactor/merge multiple sections (that don't exist in the spec if we don't pull in this PR) based on this discussion.